java/maven.indexer/src/org/netbeans/modules/maven/indexer/OnStop.java

```diff
@@ -41,7 +41,8 @@ public void run() {
                 }
                 if (t.isAlive()) {
                     LOG.warning("...hard stop required.");
-                    t.stop(new Cancellation());
+                   // t.stop(new Cancellation());
```

Comment:
An idea: Try to call it with reflection.

Reply:
for t.stop(new Cancellation())  it is okay to remove it as on  call of this method it will throw UnsupportedOperationException in jdk-8